pra_request_tracker/record_requests/migrations/0002_auto_20210630_1536.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.7 on 2021-06-30 22:36
```

Comment:
This isn't the biggest deal, but would it be possible to remove both these migration files and create a new one from scratch? That way it can all just be in the initial one